src/compute/src/logging/timely.rs

```diff
@@ -162,23 +164,29 @@ pub(super) fn construct<G: Scope<Timestamp = Timestamp>>(
             },
         );
 
-        let channels = consolidate_and_pack::<_, KeyValBatcher<_, _, _, _>, ColumnBuilder<_>, _, _>(
-            &channels,
-            TimelyLog::Channels,
-            move |((datum, ()), time, diff), packer, session| {
-                let (source_node, source_port) = datum.source;
-                let (target_node, target_port) = datum.target;
-                let data = packer.pack_slice(&[
-                    Datum::UInt64(u64::cast_from(datum.id)),
-                    Datum::UInt64(u64::cast_from(worker_id)),
-                    Datum::UInt64(u64::cast_from(source_node)),
-                    Datum::UInt64(u64::cast_from(source_port)),
-                    Datum::UInt64(u64::cast_from(target_node)),
-                    Datum::UInt64(u64::cast_from(target_port)),
-                ]);
-                session.give((data, time, diff));
-            },
-        );
+        // TODO: Consider adding eager consolidation once we deprecate columnation
```

Comment:
Is eager consolidation blocked by removing columnation? Looks like the issue is more that we don't support it with columnar?

Reply:
Fixed the comment!